lib/mixlib/install/version.rb

```diff
@@ -1,5 +1,5 @@
 module Mixlib
   class Install
-    VERSION = "0.8.0"
+    VERSION = "1.0.0"
```

Comment:
I'm wrong on this again.  Looks like the correct thing to do is leave the version untouched until it's time to release.  We should follow what chef-acceptance is doing: https://github.com/chef/chef-acceptance#releasing-chef-acceptance